apps/src/eval/eval.js

```diff
@@ -106,6 +106,27 @@ Eval.init = function(config) {
       var background = document.getElementById('background');
       background.setAttributeNS('http://www.w3.org/1999/xlink', 'xlink:href',
         skin.assetUrl('background_grid.png'));
+
+      var origin = -200;
+      var bbox, text;
+      for (var label = origin; label <= 200; label += 100) {
+        text = document.createElementNS('http://www.w3.org/2000/svg', 'text');
+        // Position text just inside the bottom right corner.
+        text.appendChild(document.createTextNode(label));
+        svg.appendChild(text);
+        bbox = text.getBBox();
+        text.setAttribute('x', label - origin - bbox.width - 3);
+        text.setAttribute('y', CANVAS_HEIGHT - bbox.height);
+        text.setAttribute('dominant-baseline', 'hanging');
```

Comment:
Hmmm, not aligned quite as nicely as I'd like, but probably good enough
![image](https://cloud.githubusercontent.com/assets/1767466/6338410/88a19892-bb63-11e4-8ef3-88691be43f60.png)